Preference Access SSN Matching/REPWRITERSPECS/RB.PREFACCESS.SSNMATCHING

```diff
@@ -99,7 +100,7 @@
 ** <MOD HISTORY>
 **   <VER 1.0.0>
 **     PROGRAMMER:  TKainz Symitar
-**     DATE:        05/03/2023
+**     DATE:        05/08/2023
 **     RELEASE:     2022.01 SP2
 **     CASE/DEFECT: N/A - Internal
 **     DETAILS:     Original Programming
```

Comment:
I think you mean to use BNOCFGFILENAME instead of BNODEBUGLFTITLE in line 286

   PRINT "Error reading CFG Letter file "+BNODEBUGLFTITLE+FORMAT(": 999",BNOERRORCODE)+
         " - "+BNOCFGFILEFERROR